### DISCUSSION TITLE
will you continue updating 1.20.1?

### DISCUSSION THREAD

**Discussion by DIREKTRYX-VyOS** [Kotori316/QuarryPlus#735] [General]
Hello, good day, it's a pleasure to be part of this community, I love this mod and I always download it when I make new worlds when the purpose of them is not precisely mining.
I would like to know if there is any news regarding support for version 1.20.1 and if it will continue to be updated.

**Comment by Kotori316** [comment]
@MORENAZI-UnUr 
I'm very sorry for late response. GitHub doesn't send me any notification for this discussion.
The 1.20.1 version will be updated if there is a bug or feature request I can implement.
